operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3351,3 +3351,6 @@ continuous_test:
 
   tolerations: []
   terminationGracePeriodSeconds: 30
+
+# -- Add dynamic manifests via values:
+extraObjects: []
```

Comment:
Please add some example like the grafana helm chart for reference

Reply:
thanks @krajorama , updated